gradle/project.gradle.kts

```diff
@@ -213,7 +214,7 @@ tasks {
             workDir.resolve("lib").deleteRecursively()
             workDir.resolve("widgets").walkBottomUp()
                     .filter { it.path.endsWith("class") }
-                    .fold(true, { res, it -> (it.delete() || !it.exists()) && res })
+                    .fold(true) { res, it -> (it.delete() || !it.exists()) && res }
```

Comment:
I think the previous version is more readable. What do you think?

Reply:
IDEA suggests it that way, so I simply obey ^^

Reply:
That is odd, because the general Kotlin rule is (was) the opposite. I'll check tomorrow. I'd really prefer to always take out lambda out of the parenthesis.

Reply:
But it is outside the parenthesis?

Reply:
I misread. We are in agreement then. All is right.